### PR TITLE
[17.0][MIG] delivery_line_sale_line_position: Migration to 17.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**
[Delivery Slip - Oscar Morgan - WH_IN_00006.pdf](https://github.com/xaviedoanhduy/stock-logistics-reporting/files/15033133/Delivery.Slip.-.Oscar.Morgan.-.WH_IN_00006.pdf)

### CODE REVIEW

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
The position does not show when Picking is Done
image

**Comment by nguyenminhchien** [comment]
this module depends on sale_order_line_position, so you can push a commit like this
https://github.com/nguyenminhchien/intrastat-extrastat/pull/4/commits/a7d489fb4e43e3a3e5daedef5ff867fedf38b9ed

**Comment by xaviedoanhduy** [comment]
Thanks for your instructions, here is the new commit to add
test-requirements.txt.
<https://github.com/xaviedoanhduy/stock-logistics-reporting/pull/2/commits/f35182d68d4a2af0e8e95c1b8a7223c3b9c107c0>

**Comment by xaviedoanhduy** [comment]
> The position does not show when Picking is Done image

I checked and there is no problem exporting that report, position not appearing because configuration group and permissions are missing, to display position when Picking is Done. You have to add group "Display Serial & Lot Number in Delivery Slips" to your user, then the product lines in the Operations tab of Picking must exist in the "Lots/Serial Numbers" view with the following menu path "Inventory/Products/Lots/Serial Numbers".

**Review by nguyenminhchien — CHANGES_REQUESTED** [review]
The checks are red.

File: test-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+odoo-addon-sale_order_line_position @ git+https://github.com/OCA/sale-reporting.git@refs/pull/261/head#subdirectory=sale_order_line_position
```
Comment (nguyenminhchien) [inline test-requirements.txt]: the pull id is 264

**Review by nguyenminhchien — APPROVED** [review]

**Pull request closed without merging**